ROM.sh

```diff
@@ -1221,6 +1221,37 @@ function build() # 4
                         ;;
                     *) echo -e "${FLD} Invalid Selection.\n" ;;
                 esac
+                # Jack cannot be disabled in N
+                # Jack workaround prompt is asked if this is set to y/Y
+                SBJK="y";
+            fi
+            if [[ "${SBJK}" == [Yy] ]] && [[ "$(free | awk '/^Mem:/{print $2}')" -lt 4096 ]]; then
```

Comment:
`free -m` didn't work
```[arvind@AspiringManjaro ~]$ free -m
free: Multiple unit options doesn't make sense.
```

`free` works properly

`-lt` should be used for comparing integers

`${2}` -> `$2` - `awk` threw an error on that